organization/_posts/2018-05-24-coordination.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "HSF Weekly Meeting #136, 24 May, 2018"
+layout: default
+---
+
+# {{page.title}}
+
+#### *Present/Contributors*: Graeme Stewart, Dario Menasce, Benedikt Hegner, Michel Jouvin, David Lange, Davide Costanzo, Torre Wenaus, Guilherme Amadio, Daniel Elvira, Andrea Sciaba, Eduardo Rodrigues, Paolo Calafiura, Pere Mato
+
+News, general matters
+=====================
+-   [EIC Software Consortium
+    Talk](https://www.jlab.org/indico/event/264/other-view?view=standard)
+    -   Well received. Discussion focused on licenses and R&D in
+        simulation.
+    -   EIC happy to give us an HSF talk in return on their activities.
+-   [IEEE eScience
+    conference](https://www.escience2018.com/), Exascale
+    Computing in High-Energy Physics track: abstract deadline extended
+    to June 18.
+-   Presentation by Benedikt at Belle II software week. Feedback about
+    where more collaboration is possible and desired. For licensing
+    just following what the community decides.
```

Comment:
my bad. Forgot to put that in initially: Copyright holder unclear. CERN was brought up as potential community-wide code owner.

Reply:
Should I add anything about KEK and DESY being other options?

Reply:
Hmm... leave that out for now.